website/content/docs/concepts/integrated-storage/autopilot.mdx

```diff
@@ -52,7 +52,7 @@ Autopilot automatically.
 
 Autopilot exposes a [configuration
 API](/vault/api-docs/system/storage/raftautopilot#set-configuration) to manage its
-behavior. Autopilot gets initialized with the following default values.
+behavior. Autopilot gets initialized with the following default values. Please note these default values are simply for reference.
```

Comment:
Maybe, it's better to phrase this as...

```suggestion
behavior. Autopilot gets initialized with the following default values. If these default values do not meet your expected autopilot behavior, don't forget to set them to your desired values.
```

Saying that the default values are "simply for reference" may send wrong message?